talpid-tunnel-config-client/examples/psk-exchange.rs

```diff
@@ -1,33 +1,39 @@
 //! Example client implementing the quantum resistant tunnel PSK exchange.
 //! Useful to test this crate's implementation.
 
+// Usage: ./psk-exchange <tuncf_server_ip> <wireguard_public_key>
```

Comment:
`s/tuncf/tuncfg`. A pretty bad abbreviation really. But it stands for "tunnel configuration".

Reply:
I figured that part out. It was more about checking whether you consistently used `tuncfg` or whether I could use Tunnel configuration instead in the error message (basically anything to begin the sentence with a capital letter and not having to resort to the weird `Tuncfg` variant.) But eventually I decided that it's not important.